server/player/src/packet_handlers/block_interaction/crafting_table.rs

```diff
@@ -0,0 +1,33 @@
+use super::InteractionHandler;
+
+use feather_core::network::packets::OpenWindow;
+use feather_core::text::TextRoot;
+use feather_server_types::{Game, Network};
+use fecs::{Entity, World};
+
+pub struct CraftingTableInteraction;
+
+impl InteractionHandler for CraftingTableInteraction {
+    fn handle_interaction(
+        &self,
+        _game: &mut Game,
+        world: &mut World,
+        player: Entity,
+        window_id: u8,
+    ) {
+        // Show the crafting table GUI for the player
+        let open_window_packet = Box::new(OpenWindow {
+            window_id,
+            window_type: String::from("minecraft:crafting_table"),
+            window_title: TextRoot::from("Crafting Table").into(),
+            number_of_slots: 0,
+            entity_id: None,
+        });
+
+        let network = world.get::<Network>(player);
+
+        network.send_boxed(open_window_packet);
```

Comment:
Call `network.send(open_window_packet)` and don't box the packet (`send_boxed` is only required when you have a `Box<dyn Packet>` and don't know the actual type.)

Reply:
Okay, thanks!